pkg/collector/corechecks/system/iostats_nix.go

```diff
@@ -108,28 +119,28 @@ func (c *IOCheck) nixIO() error {
 			aWait = float64(ioStats.ReadTime-lastIOStats.ReadTime+ioStats.WriteTime-lastIOStats.WriteTime) / diffNIO
 		}
 
-		tput := diffNIO * float64(hz)
-		util := float64(ioStats.IoTime - lastIOStats.IoTime)
+		// itv: time interval in 1/100th of a second
```

Comment:
could you please add a comment here to the relevant line(s) in `iostat.c`/`common.h` in `sysstat`? Or if you feel that's not the most clear, maybe just a quick line saying we are aligning ourselves with the metric as reported by sysstat?